exodus/search/templates/search_form.html

```diff
@@ -52,13 +55,18 @@ <h4>{% trans "Better understand" %}</h4>
     <p>{% trans "What is a tracker? And a permission? Are there any alternatives to protect yourself?" %}</p>
   </div>
 </div>
-
 <div class="row justify-content-center mb-5">
-  <div class="col-6 col-md-3 text-center mb-4">
-    <a href="{% url 'understand' %}" class="btn btn-secondary btn-block">{% trans "Let's go!" %}</a>
-  </div>
+  <div class="col-lg-8 col-sm-12">
+    <div class="row justify-content-center mb-5">
```

Comment:
We should get rid of the "mb-5" which adds an unnecessary margin

Reply:
Yeah sure, Done

Reply:
Yeah sure!